spikeinterface/curation/sortingview_curation.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from .curationsorting import CurationSorting
+
+
+def apply_sortingview_curation(
+    sorting,
+    uri,
+    exclude_labels=None,
+    include_labels=None,
+    verbose=False
+):
+    """
+    Apply curation from SortingView manual curation.
+    First, merges (if present) are applied. Then labels are loaded and units
+    are optionally filtered based on exclude_labels and include_labels.
+
+    Parameters
+    ----------
+    sorting : BaseSorting
+        The sorting object to be curated
+    uri : str
+        The URI or JOT curation link from sortingview
+    exclude_labels : list, optional
+        Optional list of labels to exclude (e.g. ["reject", "noise"]).
+        Mutually exclusive with include_labels, by default None
+    include_labels : list, optional
+        Optional list of labels to include (e.g. ["accept"]).
+        Mutually exclusive with exclude_labels,  by default None
+    verbose : bool, optional
+        If True, output is verbose, by default False
+
+    Returns
+    -------
+    sorting_curated : BaseSorting
+        The curated sorting
+    """
+    try:
+        import kachery_cloud as kcl
+    except ImportError:
+        raise ImportError(
+            "To apply a SortingView manual curation, you need to have sortingview installed: "
+            ">>> pip install sortingview"
+        )
+    curation_sorting = CurationSorting(sorting, make_graph=False, properties_policy="keep")
+
+    # get sorting view curation
+    try:
+        sorting_curation = kcl.load_json(uri=uri)
```

Comment:
This variavle name is very very ambiguous `curation_sorting` is already used for `CurationSorting`
Here it is a simple dict.
I would call it `curation_dict`

Reply:
renamed `sortingview_curation_dict`